website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -145,6 +149,104 @@ If `enable_auto_scaling` is set to `false`, then the following fields can also b
 
 ---
 
+A `kubelet_config` block supports the following:
+
+* `allowed_unsafe_sysctls` - (Optional) Specifies the allow list of unsafe sysctls command or patterns (ending in `*`). Changing this forces a new resource to be created.
+
+* `container_log_max_line` - (Optional) Specifies the maximum number of container log files that can be present for a container. must be at least 2. Changing this forces a new resource to be created.
+
+* `container_log_max_size_mb` - (Optional) Specifies the maximum size (e.g. 10Mi) of container log file before it is rotated. Changing this forces a new resource to be created.
+
+* `cpu_cfs_quota_enabled` - (Optional) Is CPU CFS quota enforcement for containers enabled? Changing this forces a new resource to be created.
+
+* `cpu_cfs_quota_period` - (Optional) Specifies the CPU CFS quota period value. Changing this forces a new resource to be created.
+
+* `cpu_manager_policy` - (Optional) Specifies the CPU Manager policy to use. Possible values are `none` and `static`, Changing this forces a new resource to be created.
+
+* `image_gc_high_threshold` - (Optional) Specifies the percent of disk usage after which image garbage collection is always run. Must be between `0` and `100`. Changing this forces a new resource to be created.
+
+* `image_gc_low_threshold` - (Optional) Specifies the percent of disk usage before which image garbage collection is never run. Must be between `0` and `100`. Changing this forces a new resource to be created.
+
+* `pod_max_pids` - (Optional) Specifies the maximum number of processes per pod. Changing this forces a new resource to be created.
+
+* `topology_manager_policy` - (Optional) Specifies the Topology Manager policy to use. Possible values are `none`, `best-effort`, `restricted` or `single-numa-node`. Changing this forces a new resource to be created.
+
+---
+
+A `linux_os_config` block supports the following:
+
+* `swap_file_size_mb` - (Optional) Specifies the size of swap file on each node in MB. Changing this forces a new resource to be created.
+
+* `sysctl_config` - (Optional) A `sysctl_config` block as defined below. Changing this forces a new resource to be created.
+
+* `transparent_huge_page_defrag` - (Optional) Specifies the Transparent Huge Page defrag configuration. Possible values are `always`, `defer`, `defer+madvise`, `madvise` and `never`. Changing this forces a new resource to be created.
```

Comment:
same here, this description doesn't make sense - presumably this "specifies the debug mode of the Transparent Huge Page"?